mask_flags.py

```diff
@@ -100,7 +100,7 @@ def run(cmd):
     cmd = prepare_subprocess_cmd(cmd)
     print("Running the following cmd:")
     for token in cmd:
```

Comment:
you can do `print(' '.join(cmd))` if you want to print without the python list. That being said, I think the list is a more correct representation, because it shows you how the subprocess module passes in the args.